### PR TITLE
Correct run-tests documentation to observed behaviour

### PR DESCRIPTION
**Pull request by wilbowma**
Addresses #135

### CODE REVIEW

File: rackunit-doc/rackunit/scribblings/ui.scrbl [hunk 1]
```diff
@@ -17,9 +17,10 @@ It is run via the @racket[run-tests] function.
          natural-number/c]{
 
 The given @racket[test] is run and the result of running it
-output to the @racket[current-output-port].  The output is
-compatable with the (X)Emacs next-error command (as used,
-for example, by (X)Emacs's compile function)
+output to the @racket[current-output-port] if all tests pass, and to
+@racket[current-error-port] when there are test failurese.
```
Comment (mflatt) [inline rackunit-doc/rackunit/scribblings/ui.scrbl:21]: Typo: "failurese".

File: rackunit-doc/rackunit/scribblings/ui.scrbl [hunk 2]
```diff
@@ -17,9 +17,10 @@ It is run via the @racket[run-tests] function.
...
+The output is compatable with the (X)Emacs next-error command (as used, for
+example, by (X)Emacs's compile function)
```
Comment (mflatt) [inline rackunit-doc/rackunit/scribblings/ui.scrbl:23]: Another nit: It looks like the sentence needs an ending `.` (and needed it before your changes).

**Review by mflatt — CHANGES_REQUESTED** [review]

**Comment by wilbowma** [comment]
I fixed those typos; I marked them as resolved, but github still shows this as unsolved and I don't know why.

**Comment by samth** [comment]
This has conflicts with the main branch, can you fix that?

**Comment by wilbowma** [comment]
I've rebased on the main branch.

**Comment by samth** [comment]
The typo noted by @mflatt seems to still be there.

**Pull request merged**